src/components/IntervalScheduleForm.vue

```diff
@@ -154,5 +155,10 @@
     @apply
     col-span-3
   }
+
+  .interval-schedule-form__column--span-4 {
+    @apply
+    col-span-4
+  }
```

Comment:
I think you could just move these two out of the columns (leaving the cols just for the two things that need cols)

The p-content should take care of the row spacing.